son-gtklic/docs/swagger.yaml

```diff
@@ -0,0 +1,547 @@
+# this is an example of the Uber API
+# as a demonstration of an API spec in YAML
+swagger: '2.0'
+info:
+  title: Sonata's License Manager
+  description: Manage licenses Sonata services.
+  version: "1"
+# the domain of the service
+host: sonata-vm1.aws.atnog.av.it.pt
+# array of all schemes that your API supports
+schemes:
+  - http
+# will be prefixed to all paths
+basePath: /api/v1
```

Comment:
That would require a change in every view(route) function.
Can you provide an example of the header that we should use?